v22.2/changefeed-sinks.md

```diff
@@ -41,6 +41,8 @@ Example of a Kafka sink URI:
 'kafka://broker.address.com:9092?topic_prefix=bar_&tls_enabled=true&ca_cert=LS0tLS1CRUdJTiBDRVJUSUZ&sasl_enabled=true&sasl_user={sasl user}&sasl_password={url-encoded password}&sasl_mechanism=SASL-SCRAM-SHA-256'
 ~~~
 
+{% include {{ page.version.version }}/misc/external-connection-kafka.md %}
```

Comment:
nit: this include comes across as a slight aside to the surrounding text: related, but not directly discussing how to make Kafka sink URIs. Perhaps if rendered as a blue note? It's somewhat of an alternative to the rest of this section: "wait don't build out that Kafta URI for your one-off query, define an external connection instead!" is kind of what we're saying. Not sure, very optional.

Reply:
No, good point. I think using the note box for now is a good idea. More sink support should be coming shortly, so with hope this is only a temporary note.